ui/components/app/modals/transaction-already-confirmed/transaction-already-confirmed.test.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import configureMockStore from 'redux-mock-store';
+import mockStore from '../../../../../test/data/mock-state.json';
+
+import {
+  renderWithProvider,
+} from '../../../../../test/jest';
+import TransactionAlreadyConfirmed from '.';
+
+const getStoreWithModalData = () => {
+  return configureMockStore()({
+    ...mockStore,
+    appState: {
+      ...mockStore.appState,
+      modal: {
+        modalState: {
+          props: {
+            originalTransactionId: 'test',
+          },
+        },
+      },
+    },
+  });
+};
+
+describe('Transaction Already Confirmed modal', () => {
+  it('should match snapshot', async () => {
+    const { container } = renderWithProvider(
```

Comment:
`baseElement` I think as it's a modal.

Reply:
Thanks for noticing this https://github.com/MetaMask/metamask-extension/pull/22943/commits/7c64152b15f6224a390e6f8200288f56fefc0011